boards/arc/em_starterkit/doc/index.rst

```diff
@@ -310,8 +310,8 @@ Release Notes
 
 The following is a list of TODO items:
 
-* :jira:`GH-2647`: Zephyr needs i-cache API (all targets)
```

Comment:
Just for my understanding, where are these ID's pointing to?

Reply:
They used to point at  https://jira.zephyrproject.org/browse/GH-2647 etc but they are all dead now.